src/plugins/plugin.ts

```diff
@@ -14,7 +14,7 @@ export { IBlock, StringOrTag };
  */
 export interface IFile {
     path: string;
-    read: () => string;
+    read(): string;
```

Comment:
well, you might argue that it signals intent: when implementing classes that extend this interface, it could suggest a bound vs unbound method:

```ts
class File implements IFile {
    read(): string { ... }
    // vs
    read: () => { ... }
}
```